app/src/main/java/org/hackillinois/android/view/ProjectAdapter.kt

```diff
@@ -7,7 +7,7 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import kotlinx.android.synthetic.main.mentor_list_item.view.*
 import org.hackillinois.android.R
-import org.hackillinois.android.model.projects.ProjectModel
+import org.hackillinois.android.database.entity.ProjectModel
 
 class ProjectAdapter internal constructor(private val mentorList: List<ProjectModel>) : RecyclerView.Adapter<ProjectAdapter.ViewHolder>() {
 
```

Comment:
Could you also rename the xml file used on line 19 to by project_list_item? Just want to be consistent everywhere